Runtime/DotNetE2EServerApp/DataObjects/TestUser.cs

```diff
@@ -15,7 +15,7 @@ public class TestUser : ITableData
 
         public string UserId { get; set; }
 
-        public string[] Identities { get; set; }
+        public string Identities { get; set; }

```

Comment:
Yeah we should revisit, but I wanted to try to minimize disruption on other client test suites.  We will see when others update them if this needs additional tweaking/redesign.